src/rhsmlib/dbus/objects/entitlement.py

```diff
@@ -30,6 +30,10 @@
 
 
 class EntitlementDBusObject(base_object.BaseObject):
+    """
+    A D-Bus object interacting with subscription-manager to list, get status
```

Comment:
should we be so specific here? if we are then we will have to change this every time we add or update the spec underneath.

Reply:
You are right. More generic comment will be better.